#160702910 user login

nwashangai opened the pull request:
#### What does this PR do?
it adds user login functionality
#### Description of Task to be completed?
* create login endpoint
* validate inputs
* write test for login endpoint
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
[#160702910](https://www.pivotaltracker.com/story/show/160702910)
#### Screenshots (if appropriate)
* None
#### Questions:
* None

nwashangai committed:
ft- user login
    - create login endpoint
    - validate inputs
    - write test for login edpoint
    [Finishes #160702910]

nwashangai committed:
ft: place order
    - create endpoint for placing order
    - vaidate ndpoint
    - write test for place order endpoint
    [Finishes #160702928]

dogoyaro commented:
There should be changes added to your README.md file about how to hit this endpoint.

nwashangai committed:
ft: get order history for a particular user
    - crreate endpoint to get order history of user
    - write test for the endpoint
    [Finishes #160702946]

nwashangai committed:
Merge pull request #36 from nwashangai/ft-user-order-history-2197024

nwashangai committed:
ft: Admin can add food
    - create endpoint to add food
    - validate user inputs
    - write test cases for this endpoint
    [Finishes #160703460]

nwashangai committed:
ft: get food menu
    - create endpoint to get food menu
    - write test for this endpoint
    [Finishes #160703275]

nwashangai committed:
ft: update order status
    - create endpiont to update status
    - write test cases for this endpoint
    [Finishes #160703227]

nwashangai committed:
ft: admin gets specific order
    - create endpoint to get specific order
    - write test cases for this endpoint
    [Finishes #160703172]

nwashangai committed:
ft: admin gets all orders
    - create endpoint to get all orders
    - write test cases for this endpoint
    [Finishes #160703159]

nwashangai committed:
ft: adds update menu item functionalities
    - create endpoint to update menu item
    - write test cases for this endpoint
    [Finishes #160703481]

nwashangai committed:
Merge pull request #42 from nwashangai/ft-update-menu-item-2197024

#160703481 adds update menu item functionalities

nwashangai committed:
Merge pull request #41 from nwashangai/ft-get-all-orders-2197024

#160703159 admin gets all orders

nwashangai committed:
Merge pull request #40 from nwashangai/ft-get-specific-2197024

#160703172 admin gets specific order

nwashangai committed:
Merge pull request #39 from nwashangai/ft-update-status-2197024

nwashangai committed:
Merge pull request #38 from nwashangai/ft-get-menu-2197024

#160703275 get food menu

nwashangai committed:
Merge pull request #37 from nwashangai/ft-add-food-2197024

nwashangai committed:
Merge pull request #35 from nwashangai/ft-place-order-2197024
